pom.xml

```diff
@@ -92,6 +93,11 @@
             <version>${auto-service.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
It looks like some of the dependencies in this project are slightly out of date. 

When running `mvn versions:display-dependency-updates` to check which new versions are available, the following new versions are indicated. Some of these reference dependencies in the server core, which can be ignored (eg. junit, assertj, auto-value), but I think some of others can be updated.

> [INFO] The following dependencies in Dependencies have newer versions:
> [INFO]   com.google.auto.service:auto-service .............. 1.0-rc4 -> 1.0-rc6
> [INFO]   com.google.auto.value:auto-value ........................ 1.6.2 -> 1.7
> [INFO]   com.google.protobuf:protobuf-java-util ............... 3.6.1 -> 3.11.0
> [INFO]   io.pkts:pkts-core ..................................... 3.0.3 -> 3.0.5
> [INFO]   junit:junit ........................................ 4.12 -> 4.13-rc-1
> [INFO]   org.assertj:assertj-core ............................ 3.10.0 -> 3.14.0

Reply:
Good catch! Will do. Thanks!